### DISCUSSION TITLE
Does module order in the GUI matter?

### DISCUSSION THREAD

**Discussion by ssagerian** [AllskyTeam/allsky-modules#99] [General]
Hello,
just starting to use the modules. Does the order in which they appear in the GUI matter?
Just thinking that the clear sky one needs to run first in order for the meteor module to know about the clear sky and lastly the module overlay. If they ran in the opposite order would things applied by the overlay possibly affect the meteor detection? thanks!
Great job
Steve
![image](https://github.com/AllskyTeam/allsky-modules/assets/3128943/085bb748-5f7c-4d81-9fe4-cb83bd139fda)

**Comment by Alex-developer** [comment]
Yes the order does matter. Generally the modules should run in the order you need the data, if that makes sense.

Your description is spot on in terms of the order :-)

Cheers

Alex
